### ISSUE TITLE
Problem picking aligment points in Blender 2.8

### ISSUE THREAD

**Issue by leonlazic** [patmo141/object_alignment#19]
Hi I am experiencing an issue when I try to pick points for aligment.
When I select both the base object an the align object and click on "Align: Picked Points" Button I get this error:

`Traceback (most recent call last):
  File "/Users/leonlazic/Library/Application Support/Blender/2.80/scripts/addons/patmo141-object_alignment-2f5bc25/operators/align_pick_points.py", line 241, in invoke
    ob.select = False
AttributeError: 'Object' object has no attribute 'select'`

Align Object Version: 0.2.0
OS: Mac OS
Blender Version: 2.80 2019-07-29

Best Regards, Leon

**Comment by patmo141** [comment]
you need to download the 2.80 branch

**Comment by patmo141** [comment]
https://github.com/patmo141/object_alignment/tree/b_280

**Comment by leonlazic** [comment]
I have downloaded the 2.8 branch. Otherwise I wouldn’t expect it to work

**Comment by patmo141** [comment]
you may nee to clear out the cached addon files.  As you can see, the error mentions line 241.  In master branch, the line matches the error you have.  In b_280 branch, the line is completely different because of the additional gpu code I had to add at the top of the file.

https://github.com/patmo141/object_alignment/blob/master/operators/align_pick_points.py#L241

https://github.com/patmo141/object_alignment/blob/b_280/operators/align_pick_points.py#L241

**Comment by patmo141** [comment]
just make sure you click this little dropdown before the "clone or download" button.

![image](https://user-images.githubusercontent.com/2917743/64070134-a4f83980-cc28-11e9-950a-1126b1aec5bc.png)

and the download should be named the following  "object_alignment-b_280.zip"

**Comment by leonlazic** [comment]
Thanks. 
But I know now what happened. I downloaded the 2.8 branch already the first time but right after installing the addon (before checking if it actually works) I clicked on "check for update" and it said that the 0.3.0 version is available. 
So I said " Cool newer version probably even some early bugs fixed already. I'll update right away..." And after the update I got that error and even some other ones.
But now I figured that the update probably doesn't link to the correct branch. Thanks again for your help.

Best regards, Leon
